CHANGES.md

```diff
@@ -2,10 +2,19 @@
 
 ### The Next Release
 
-- Make `version.js` more detailed. It now uses format `${date}-${TerriaMap version}-${TerriaJS version}-${TerriaMap commit hash}`
-- Also write out new `wwwroot/version.json` file - for example `{"date":"2022-11-11","terriajs":"8.2.21","terriamap":"0.0.6","terriamapCommitHash":"2d4d8d7d","hasLocalModifications":true}`
```

Comment:
I think these changes are still relevant. People using the TerriaMap docker image may be interested in version.json

Reply:
My bad. Fixed!